template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/StarterWeb-CSharp/Data/Migrations/ApplicationDbContextModelSnapshot.cs

```diff
@@ -0,0 +1,245 @@
+﻿using System;
+#if (UseLocalDB)
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+#endif
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Infrastructure;
+using Microsoft.EntityFrameworkCore.Metadata;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace Company.WebApplication1.Data.Migrations
+{
+    [DbContext(typeof(ApplicationDbContext))]
+    partial class ApplicationDbContextModelSnapshot : ModelSnapshot
+    {
+        protected override void BuildModel(ModelBuilder modelBuilder)
+        {
+            modelBuilder
+#if (UseLocalDB)
+                .HasAnnotation("ProductVersion", "1.0.0-rc3")
+                .HasAnnotation("SqlServer:ValueGenerationStrategy", SqlServerValueGenerationStrategy.IdentityColumn);
+#else
+                .HasAnnotation("ProductVersion", "1.0.2");
+#endif
+
+            modelBuilder.Entity("Microsoft.AspNetCore.Identity.EntityFrameworkCore.IdentityRole", b =>
```

Comment:
Need to remove the EntityFrameworkCore in the namespace for all of these entities in the migration, i.e.:

"Microsoft.AspNetCore.Identity.IdentityRole"